tests/python/unittest/test_image.py

```diff
@@ -110,9 +110,9 @@ def test_resize_short(self):
             for _ in range(3):
                 new_size = np.random.randint(1, 1000)
                 if h > w:
-                    new_h, new_w = new_size * h / w, new_size
+                    new_h, new_w = int(new_size * h / w), new_size
```

Comment:
The float number comes from the division, which in python3 is float division by default. You can just replace `/` with `//` to fix.

Reply:
Looks more clear and resonable. I updated.